plugins/modules/helm_cli_info.py

```diff
@@ -0,0 +1,290 @@
+  helm_cli_info:
+    name: test
+    tiller_namespace: helm
+
+# With Helm 3
+- name: Deploy latest version of Grafana chart inside monitoring namespace
+  helm_cli_info:
+    name: test
+    release_namespace: monitoring
+'''
+
+RETURN = """
+status:
+  type: complex
+  description: A dictionary of status output
+  returned: only when release exists
+  contains:
+    appversion:
+      type: str
+      returned: always
+      description: Version of app deployed
+    chart:
+      type: str
+      returned: always
+      description: Chart name and chart version
+    name:
+      type: str
+      returned: always
+      description: Name of the release
+    namespace:
+      type: str
+      returned: always
+      description: Namespace where the release is deployed
+    revision:
+      type: str
+      returned: always
+      description: Number of time where the release has been updated
+    status:
+      type: str
+      returned: always
+      description: Status of release (can be DEPLOYED, FAILED, ...)
+    updated:
+      type: str
+      returned: always
+      description: The Date of last update
+    values:
+      type: str
+      returned: always
+      description: Dict of Values used to deploy
+"""
+
+try:
+    import yaml
+    HAS_YAML = True
+except ImportError:
+    HAS_YAML = False
+
+from ansible.module_utils.basic import AnsibleModule
+
+module = None
+is_helm_2 = True
+
+
+# get Helm Version
+def get_helm_client_version(command):
+    is_helm_2_local = True
+    version_command = command + " version --client --short"
+    rc, out, err = module.run_command(version_command)
+
+    if not out.startswith('Client: v2'):
+        is_helm_2_local = False
+        # Fallback on Helm 3
+        version_command = command + " version --short"
+        rc, out, err = module.run_command(version_command)
+
+        if rc != 0:
+            module.fail_json(
+                msg="Failure when executing Helm command. Exited {0}.\nstdout: {1}\nstderr: {2}".format(rc, out, err),
+                command=version_command
+            )
+
+    elif rc != 0:
+        module.fail_json(
+            msg="Failure when executing Helm command. Exited {0}.\nstdout: {1}\nstderr: {2}".format(rc, out, err),
+            command=version_command
+        )
+
+    return is_helm_2_local
+
+
+# Get Values from deployed release
+def get_values(command, release_name, release_namespace):
+    get_command = command + " get values --output=yaml " + release_name
+
+    if not is_helm_2:
+        get_command += " --namespace=" + release_namespace
+
+    rc, out, err = module.run_command(get_command)
+
+    if rc != 0:
+        module.fail_json(
+            msg="Failure when executing Helm command. Exited {0}.\nstdout: {1}\nstderr: {2}".format(rc, out, err),
+            command=get_command
+        )
+
+    # Helm 3 return "null" string when no values are set
+    if not is_helm_2 and out.rstrip("\n") == "null":
+        return yaml.safe_load('{}')
```

Comment:
just a nit, but why not just return `{}` directly?

Reply:
It was a temporary fix for helm3 during alpha but the issue is still here
Your suggestion is better, i will change that
Thanks